charts/crowdsec/templates/appsec-deployment.yaml

```diff
@@ -10,17 +10,17 @@ metadata:
     k8s-app: {{ .Release.Name }}
     type: appsec
     version: v1
-  {{- if .Values.appsec.deployAnnotations }}
```

Comment:
Is it ok for you if we keep it as is. It will help on backward compat. 
If we change this right now, maybe some users are already working with this deployAnnotations and it can break their chart upgrade.

Reply:
Sure, preventing unbreaking changes is probably a good idea :)